glide.yaml

```diff
@@ -5,8 +5,10 @@ import:
   - base58
 - package: github.com/FactomProject/cli
 - package: github.com/FactomProject/factom
-  version: v1-rollup-testing
```

Comment:
We should not pin the glide.yaml to a branch. Let's remove this line all together